### PR TITLE
News image changes

### PR DESCRIPTION
**Pull request by drewvolz**
These two commits change the News views:

- Aligns images to left side of view
- Rounds the image thumbnail corners
- Uses a placeholder for blank images
- Adds logos for the placeholders
- Shortens the separator line length

Politicole | Oleville | KSTO
--|--|--
politicole| oleville | ksto

### CODE REVIEW

File: source/views/news/news-list.js [hunk 1]
```diff
@@ -30,8 +31,10 @@ export class NewsList extends React.Component {
     })
   }
 
+  renderSeparator = () => <ListSeparator spacing={{left: 101}} />
+
   renderItem = ({item}: {item: StoryType}) =>
-    <NewsRow onPress={this.onPressNews} story={item} />
+    <NewsRow onPress={this.onPressNews} story={item} blank={this.props.thumb} />
```
Comment (hawkrives) [inline source/views/news/news-list.js]: Why is this new prop called `blank`? Please change it to `thumb` or `thumbnail`.

File: source/views/news/news-row.js [hunk 2]
```diff
@@ -35,8 +36,9 @@ export class NewsRow extends React.PureComponent {
 
 const styles = StyleSheet.create({
   image: {
-    marginLeft: 8,
-    height: 50,
-    width: 50,
+    borderRadius: 5,
```
Comment (hawkrives) [inline source/views/news/news-row.js:39]: :+1: on the `borderRadius`

File: source/views/news/news-container.js [hunk 3]
```diff
@@ -17,6 +17,7 @@ export default class NewsContainer extends React.Component {
     query?: Object,
     embedFeaturedImage?: boolean,
     mode: 'rss' | 'wp-json',
+    thumb: number,
```
Comment (hawkrives) [inline source/views/news/news-container.js]: <abbr title="bikeshed time">🚲 🏠 ⌚️</abbr>   – `thumb` or `thumbnail`?

**Review by hawkrives — CHANGES_REQUESTED** [review]
Overall, 👍!

- … can we get the new thumbnails to be all the same size?

File: source/views/news/news-container.js (hunk 3 above)
Comment (drewvolz) [reply source/views/news/news-container.js]: If you'd prefer thumbnail, that's fine.

**Comment by drewvolz** [comment]
> can we get the new thumbnails to be all the same size?

@hawkrives I've resized the circular images to all be 90x90. As far as the stolaf logo... I've resized it to be 77x90. I do not want to turn a more naturally rectangular shape into a squashed square.

Okay? I think I've satisfied the rest of your requests!

**Review by hawkrives — APPROVED** [review]

**Comment by hawkrives** [comment]
I just want to see if me poking Danger was enough to fix it.

**Comment by drewvolz** [comment]
Danger is still having a bad day.

**Comment by hawkrives** [comment]
We'll merge it anyway; it's probably just something with this particular message.

Would you mind making an issue, assigning it to me, and linking to the danger comment, please, so I can look into it later?

**Pull request merged**